docs/jwst/outlier_detection/outlier_detection_spec.rst

```diff
@@ -1,50 +1,24 @@
 .. _outlier-detection-spec:
 
-Outlier Detection for Slit-like Spectroscopic Data
-==================================================
+Slit-like Spectroscopic Data
+============================
 
 This module serves as the interface for applying ``outlier_detection`` to slit-like
-spectroscopic observations.  The code implements the
-basic outlier detection algorithm used with HST data, as adapted to JWST
-spectroscopic observations.
+spectroscopic observations. The algorithm is very similar to the
+:ref:`imaging algorithm <outlier-detection-imaging>`, and much of the same code is used.
+Please refer to those docs for more information.
+A :ref:`Stage 3 association <asn-level3-techspecs>`,
+which is loaded into a :py:class:`~jwst.datamodels.ModelContainer` object,
+serves as the input and output to this step, and the :py:class:`~jwst.datamodels.ModelContainer`
+is converted into a :py:class:`~jwst.datamodels.ModelLibrary` object where processing is
+shared with the imaging mode.
 
-Specifically, this routine performs the following operations (modified from the
-:ref:`Default Outlier Detection Algorithm <outlier-detection-imaging>` ):
+This routine performs identical operations to the imaging mode, with the following exceptions:
 
```

Comment:
This will need an addition for the recently merged PR #8828 - error handling for thresholding is now different for imaging and spectral data.